### ISSUE TITLE
Unable to build Radar XCFramework on Xcode 14 via Carthage

### ISSUE THREAD

**Issue by kunal30xi** [radarlabs/radar-sdk-ios#205]
## Summary
Receiving the following error message 'Unhandled/unexpected case where no run destinations were produced' when building the Radar SDK with Xcode 14. Building with Xcode 13.x works just fine.

## Steps to reproduce
Executing Carthage via fastlane
```
        carthage(
            command: 'bootstrap',
            use_binaries: false,
            use_xcframeworks: true,
            platform: 'iOS',
            log_path: "#{Dir.pwd}/carthage.log"
        )
```

## OS version
Mac OS Monterey 12.6

## SDK installation method
Carthage 0.38.0

## SDK version
Radar SDK 3.5.3

## Other information

Console output attached:
```
A shell task (/usr/bin/xcrun xcodebuild -project /Users/<username>/Git/CodeRepo/Carthage/Checkouts/radar-sdk-ios/RadarSDK.xcodeproj -scheme Library -configuration Release CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -showBuildSettings -skipUnavailableActions) failed with exit code 6:
2022-09-13 11:13:50.869 xcodebuild[19088:137663] [MT] DVTAssertions: ASSERTION FAILURE in /System/Volumes/Data/SWE/Apps/DT/BuildRoots/BuildRoot2/ActiveBuildRoot/Library/Caches/com.apple.xbs/Sources/IDEFrameworks/IDEFrameworks-21335/IDEFoundation/Execution/RunDestinations/IDERunDestinationCLI.m:566
Details:  Unhandled/unexpected case where no run destinations were produced.
Object:   <IDERunDestinationCLI>
Method:   +resolveRunDestinationsWithWorkspace:scheme:buildAction:schemeCommand:schemeTask:destinationSpecifications:architectures:timeout:runDestinationManager:deviceManager:fallbackPreferredSDK:fallbackPreferredArchitectures:skipUnsupportedDestinations:shouldSkipRunDestinationValidation:didDisambiguate:disambiguatedMatches:disambiguatedMatchesDescription:error:
Thread:   <_NSMainThread: 0x6000001781c0>{number = 1, name = main}
Hints: 

Backtrace:
  0   -[DVTAssertionHandler handleFailureInMethod:object:fileName:lineNumber:assertionSignature:messageFormat:arguments:] (in DVTFoundation)
  1   _DVTAssertionHandler (in DVTFoundation)
  2   _DVTAssertionFailureHandler (in DVTFoundation)
  3   _sortDevicesForDisplay (in IDEFoundation)
  4   -[Xcode3CommandLineBuildTool _resolveRunDestinationsForBuildAction:] (in Xcode3Core)
  5   -[Xcode3CommandLineBuildTool _resolveInputOptionsWithTimingSection:] (in Xcode3Core)
  6   -[Xcode3CommandLineBuildTool run] (in Xcode3Core)
  7   XcodeBuildMain (in libxcodebuildLoader.dylib)
  8   start (in dyld)
```

**Comment by jrtibbetts** [comment]
I published [release 3.5.4-beta.1](https://github.com/radarlabs/radar-sdk-ios/releases/tag/3.5.4-beta.1) to fix this. Please give it a try by specifying `3.5.4-beta.1` for the `radar-sdk-ios` dependency in your `Cartfile`.

**Comment by samfader** [comment]
@kunal30xi As you can see in the mentioned issue above, we're working on an official release. I'd subscribe to that issue for updates, and we'll also keep you posted.
